architecture/Solution Architecture.md

```diff
@@ -0,0 +1,314 @@
+# COVID-19 Notification App - Solution Architecture
+
+## Baseline for the "Proof of Concept"
+
+**Version:** 0.6 (28/5/2020)
+
+**Status: ** Draft
+
+# Introduction
+
+The ministry of VWS is currently investigating if and how a contract tracing app could be used to help with the current corona crisis. As there are significant technical, privacy, epidemiological and behaviour-science related challenges this will initially be a "Proof of Concept" (PoC) only.
+
+This document describes the functional and technical architecture of this PoC. Note that, as part of the Dutch Covid19 response, there are also several other workstreams happening in parallel (such as for example investigating a follow up, post infection, support app). 
+
+The requirements document received from the Dutch health authority GGD are leading in the decisions in this solution architecture. The original requirements document can be found at [https://www.rijksoverheid.nl/binaries/rijksoverheid/documenten/publicaties/2020/05/19/programma-van-eisen/20200519+Programma+van+Eisen+def.pdf](https://www.rijksoverheid.nl/binaries/rijksoverheid/documenten/publicaties/2020/05/19/programma-van-eisen/20200519+Programma+van+Eisen+def.pdf)
+
+**This document only details the proof of concept for exposure notification.**
+
+# Table of contents
+
+[[TOC]]
```

Comment:
This won't work out of the box on Github, see [your current markdown online](https://github.com/ijansch/nl-covid19-notification-app-coordination/blob/initial_version/architecture/Solution%20Architecture.md#table-of-contents).

Reply:
Thanks, fixed!